pep-0710.rst

```diff
@@ -171,16 +171,146 @@ from the installer's cache.
 Backwards Compatibility
 =======================
 
-Since this PEP specifies a new file in the ``.dist-info`` directory, there are
-no backwards compatibility implications to consider in the ``provenance_url.json``
-file itself. Also, this proposal does not make any changes to the
-``direct_url.json`` described in :pep:`610` and
-:ref:`its corresponding canonical PyPA spec <direct-url>`.
+Following the :ref:`packaging:recording-installed-packages` specification,
+installers may keep additional installer-specific files in the ``.dist-info``
+directory.  To make sure this PEP does not cause any backwards compatibility
+issues, there was conducted a research whether introducing the
+``provenance_url.json`` file in the ``.dist-info`` directory is feasible and
+does not cause any backwards compatibility issues in the Python packaging
+ecosystem.
+
+A specification for :ref:`binary-distribution-format` lists files that can be
+present in the ``.dist-info`` directory. None of these file names collide with
+the proposed ``provenance_url.json`` file from this PEP.
+
+Presence of provenance_url.json in installers and libraries
+-----------------------------------------------------------
```

Comment:
I appreciate the thoroughness here; I would suggest, though, as it goes into a lot of comprehensive detail and the key point can ultimately be summarized in one sentence, that none of the tools surveyed currently use or would conflict with a `provenance_url.json` fie in the `.dist-info`, that this be moved to an appendix (between the `Open Issues` and the `Acknowledgements`), as these types of surveys generally go in other PEPs (don't forget to bump the section underbars up one level):

```rst
.. _710-tool-survey:

Appendix: Survey of installers and libraries
============================================

pip
---
```

 and then referenced in the suggestion below/paragraph above with a short summary (see that suggestion).

Reply:
Thanks, adjusted in [6c9e95c](https://github.com/python/peps/pull/3076/commits/6c9e95cae8854565e7f4519586e009c6058a8e92).